### ISSUE TITLE
Cannot upload webm's

### ISSUE THREAD

**Issue by mysteriomask** [grzegorznittner/chanu#458]
I am not sure if I am doing anything wrong or not but thought this should come to your attention.  But it seems that when I try and upload a webm video to /gif/  and /wsg/.  It removes the webms from my phones gallery so that I cannot select them for uploading.

**Comment by johnarleyburns** [comment]
I don't think it has webm as a supported upload type, that's the reason.

**Comment by London1332** [comment]
Does anyone know if this feature will ever be added? Chanu is my favorite 4 Chan app, but this feature is pretty essential for many of the boards. Is it really that difficult to add?

**Comment by johnarleyburns** [comment]
It will do webm download but not upload.

**Comment by London1332** [comment]
Indeed. I've been using the app for years. The question is if it ever will.

**Comment by johnarleyburns** [comment]
We've opened sourced it so hopefully we can find someone to implement it.

**Comment by Greznor** [comment]
I've taken a stab at this without any luck. I'll try again soon.
